Framework/Algorithms/src/CreatePSDBleedMask.cpp

```diff
@@ -30,8 +30,7 @@ using DataObjects::MaskWorkspace_sptr;
 //----------------------------------------------------------------------
 
 /// Default constructor
-CreatePSDBleedMask::CreatePSDBleedMask()
-    : m_maxRate(0.0), m_numIgnoredPixels(0), m_isRawCounts(false) {}
+CreatePSDBleedMask::CreatePSDBleedMask() {}
```

Comment:
This change will likely create coverity warnings/errors. Initializer lists are a good way of setting default values.